modules/gapi/include/opencv2/gapi/imgproc.hpp

```diff
@@ -187,6 +187,26 @@ namespace imgproc {
             return in.withType(CV_8U, 1);
         }
     };
+
+    G_TYPED_KERNEL(GBayerGR2RGB, <cv::GMat(cv::GMat)>, "org.opencv.imgproc.colorconvert.bayergr2rgb") {
+        static cv::GMatDesc outMeta(cv::GMatDesc in) {
+            return in.withType(CV_8U, 3);
+        }
+    };
+
+    G_TYPED_KERNEL(GRGB2HSV, <cv::GMat(cv::GMat)>, "org.opencv.imgproc.colorconvert.rgb2hsv") {
+        static cv::GMatDesc outMeta(cv::GMatDesc in) {
+            return in.withType(CV_8U, 3);
+        }
+    };
```

Comment:
In this case meta doesn't change, right? Both RGB and HSV are 8uC3.

Reply:
Fixed